frontend/test/metabase/scenarios/visualizations/waterfall.cy.spec.js

```diff
@@ -0,0 +1,164 @@
+import { signInAsNormalUser, restore } from "__support__/cypress";
+
+describe("scenarios > visualizations > waterfall", () => {
+  beforeEach(() => {
+    restore();
+    signInAsNormalUser();
+  });
+
+  it("should work with ordinal series", () => {
+    cy.visit("/question/new");
+    cy.contains("Native query").click();
+    cy.get(".ace_content").type(
+      "select 'A' as product, 10 as profit union select 'B' as product, -4 as profit",
+    );
+    cy.get(".NativeQueryEditor .Icon-play").click();
+    cy.contains("Visualization").click();
+    cy.get(".Icon-waterfall").click();
+
+    cy.get(".Visualization .x-axis-label").within(() => {
+      cy.findByText("PRODUCT");
+    });
+    cy.get(".Visualization .y-axis-label").within(() => {
+      cy.findByText("PROFIT");
+    });
+    cy.get(".Visualization .axis.x").within(() => {
+      cy.findByText("Total");
+    });
+  });
+
+  it("should work with quantitative series", () => {
+    cy.visit("/question/new");
+    cy.contains("Native query").click();
+    cy.get(".ace_content").type(
+      "select 1 as xx, 10 as yy union select 2 as xx, -2 as yy",
+    );
+    cy.get(".NativeQueryEditor .Icon-play").click();
+    cy.contains("Visualization").click();
+    cy.get(".Icon-waterfall").click();
+
+    cy.contains("Select a field").click();
+    cy.get(".List-item")
+      .first()
+      .click(); // X
+    cy.contains("Select a field").click();
+    cy.get(".List-item")
+      .last()
+      .click(); // Y
+
+    cy.get(".Visualization .x-axis-label").within(() => {
+      cy.findByText("XX");
+    });
+    cy.get(".Visualization .y-axis-label").within(() => {
+      cy.findByText("YY");
+    });
+    cy.get(".Visualization .axis.x").within(() => {
+      cy.findByText("Total");
+    });
+  });
+
+  it("should work with time-series data", () => {
+    cy.visit("/");
+    cy.findByText("Ask a question").click();
+    cy.findByText("Custom question").click();
+    cy.findByText("Sample Dataset").click();
+    cy.findByText("Orders").click();
```

Comment:
`openOrdersTable({ mode: "notebook" })`

Of course, import openOrdersTable before this :)

Reply:
Thanks! I've adopted this syle.